tsconfig.json

```diff
@@ -1,13 +1,12 @@
 {
-    "include": ["src", "types"],
+    "include": ["src"],
     "compilerOptions": {
         "module": "esnext",
         "lib": ["dom", "esnext"],
         "importHelpers": true,
         "declaration": true,
-        "declarationDir": "dist/types",
+        "declarationDir": "types",
         "sourceMap": true,
-        "rootDir": "./src",
```

Comment:
Killed this as you've already scoped it in the `include` at the top, making the root the same as the location of this `tsconfig` file.